doc/nrf/app_dev/device_guides/thingy91x/thingy91x_features.rst

```diff
@@ -0,0 +1,21 @@
+.. _thingy91x_features:
+
+Features of Thingy:91 X
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Nordic Thingy:91 X is an advanced version of the Thingy:91.
```

Comment:
Thingy:91 and THingy:91 X are two destinct products, so I dont think we should state that the Thingy:91 X is an advanced version of the Thingy:91. Suggest replacing the sentence with the following:

"Nordic Thingy:91 X is a battery-operated prototyping platform for cellular IoT. "